crates/orchestrator/src/jobs/state_update_job/kzg.rs

```diff
@@ -0,0 +1,135 @@
+use crate::config::config;
+use crate::jobs::state_update_job::CURRENT_PATH;
+use alloy::eips::eip4844::BYTES_PER_BLOB;
+use c_kzg::{Blob, Bytes32, KzgCommitment, KzgProof, KzgSettings};
+use color_eyre::eyre::eyre;
+use lazy_static::lazy_static;
+
+lazy_static! {
+    pub static ref KZG_SETTINGS: KzgSettings = KzgSettings::load_trusted_setup_file(
+        CURRENT_PATH.join("src/jobs/state_update_job/trusted_setup.txt").as_path()
+    )
+    .expect("Error loading trusted setup file");
+}
+
+pub const BLOB_DATA_FILE_NAME: &str = "blob_data.txt";
+pub const X_0_FILE_NAME: &str = "x_0.txt";
+
+/// Build KZG proof for a given block
+pub async fn build_kzg_proof(blob_data: Vec<u8>, x_0_value: Bytes32) -> color_eyre::Result<(Vec<u8>, KzgProof)> {
```

Comment:
the build proof part needs to be inside the settlement client. celestia, for example, will have a completely different way of building the proof

Reply:
that's fine, the config is available everywhere in the app so it can access the storage client right?